src/modules/servers/application/use-cases/get-server-by-ilo-ip.use-case.spec.ts

```
@@ -0,0 +1,122 @@
+        new NotFoundException('Server with iLO IP 192.168.1.100 not found'),
+      );
+
+      expect(serverRepository.findByIloIp).toHaveBeenCalledWith('192.168.1.100');
+    });
+
+    it('should handle different iLO IP formats', async () => {
+      const differentIPs = [
+        '10.0.0.1',
+        '172.16.0.1',
+        '192.168.255.255',
+      ];
```

ESLint

[error] Replace `⏎········'10.0.0.1',⏎········'172.16.0.1',⏎········'192.168.255.255',⏎······` with `'10.0.0.1',·'172.16.0.1',·'192.168.255.255'`

(prettier/prettier)